src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: OpenTelemetry data in New relic
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - OTLP
+metaDescription: Overview of OpenTelemetry data in New RElic
+freshnessValidatedDate: never
+redirects:
+ - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-alerts/
+ - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-data-explorer-query-builder/
+---
+
+OpenTelemetry has a rich ecosystem of instrumentation and tooling which can be integrated with New Relic via [OTLP](https://opentelemetry.io/docs/specs/otlp/). This documentation describes how New Relic receives, processes, and ingests OTLP data. See [Get started with OpenTelemetry and New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-get-started-intro/) for information on common integration patterns.
+
+[New Relic OTLP endpoint](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp/) describes OTLP configuration requirements relevant for any OpenTelemetry integration with New Relic, along with [troubleshooting](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp-troubleshooting/) information.
+
+OTLP is divided into trace, metric, and log signals, all of which include the concept of a resources. See the following pages for details on how each is processed and ingested:
+
+* [OpenTelemetry traces in New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-traces/)
+* [OpenTelemetry metrics in New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-metrics/)
+* [OpenTelemetry logs in New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs/)
+* [OpenTelemetry resources in New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources/)
+
+All OTLP data is ingested into the [New Relic platform](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-platform/), which includes many powerful general purpose observability tools to help you identify and diagnose problems, including:
+
+* [Alerts](/docs/alerts-applied-intelligence/overview/): get informed when a signal exceeds a threshold.
```

Comment:
```suggestion
* [Alerts](/docs/alerts-applied-intelligence/overview/): Get informed when a signal exceeds a threshold.
```

Reply:
@Rob-S , we already spoke about this issue and have decided to hold off on changing the URLs until the final yml structure has been confirmed.